src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -52,10 +48,60 @@ You cannot use LDR on a table with a schema that contains the following:
 - [Partial indexes]({% link {{ page.version.version }}/partial-indexes.md %}) and [hash-sharded indexes]({% link {{ page.version.version }}/hash-sharded-indexes.md %})
 - Indexes with a [virtual computed column]({% link {{ page.version.version }}/computed-columns.md %})
 - Composite types in the [primary key]({% link {{ page.version.version }}/primary-key.md %})
+- [Foreign key]({% link {{ page.version.version }}/foreign-key.md %}) dependencies
 
 For more details, refer to the LDR [Known limitations]({% link {{ page.version.version }}/logical-data-replication-overview.md %}#known-limitations).
 
-When you run LDR in [`immediate` mode](#modes), you cannot replicate a table with [foreign key constraints]({% link {{ page.version.version }}/foreign-key.md %}). In [`validated` mode](#modes), foreign key constraints **must** match. All constraints are enforced at the time of SQL/application write.
+LDR does not support replicating a table with [foreign key constraints]({% link {{ page.version.version }}/foreign-key.md %}). 
+
+#### Unique secondary indexes
+
+When the destination table includes unique [secondary indexes]({% link {{ page.version.version }}/schema-design-indexes.md %}), it can cause rows to enter the [_dead letter queue_ (DLQ)]({% link {{ page.version.version }}/manage-logical-data-replication.md %}). The two clusters in LDR operate independently, so writes to one cluster can conflict with writes to the other.
+
+If the application modifies the same row in both clusters, LDR resolves the conflict using _last write wins_ (LWW) conflict resolution. [`UNIQUE` constraints]({% link {{ page.version.version }}/unique.md %}) are validated locally in each cluster, therefore if a replicated write violates a `UNIQUE` constraint on the destination cluster (possibly because a conflicting write was already applied to the row) the replicating row will be applied to the DLQ. 
+
+For example, consider a table with a unique `name` column where the following operations occur in this order in a source and destination cluster running LDR:
+
+On the **source cluster**:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+-- writes to the source table
+INSERT INTO city (1, nyc); -- timestamp 1
+UPDATE city SET name = 'philly' WHERE id = 1; -- timestamp 2
+INSERT INTO city (100, nyc); -- timestamp 3
+~~~
+
+LDR replicates the write to the **destination cluster**: 
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+-- replicates to the destination table
+INSERT INTO city (100, nyc); -- timestamp 4
+~~~
+
+_Timestamp 5:_ Range containing primary key `1` on the destination cluster is unavailable for a few minutes due to a network partition.
```

Comment:
First mention of range in this section could link to https://www.cockroachlabs.com/docs/v25.2/architecture/glossary.html#range

Reply:
'network partition' could link to https://www.cockroachlabs.com/docs/v25.2/cluster-setup-troubleshooting#network-partition